hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/ethereum/HelloWorldEthereumSuite.java

```diff
@@ -469,6 +482,104 @@ private HapiSpec contractCreateWithConstructorArgs() {
                                 .has(accountWith().nonce(1L)));
     }
 
+    private static final String JUST_SEND_CONTRACT = "JustSend";
+
+    private static final String SEND_TO = "sendTo";
+
+    HapiSpec topLevelBurnToZeroAddressReverts() {
+        final var ethBurnAddress = new byte[20];
+        return defaultHapiSpec("TopLevelBurnToZeroAddressReverts")
+                .given(
+                        newKeyNamed(SECP_256K1_SOURCE_KEY).shape(SECP_256K1_SHAPE),
+                        cryptoCreate(RELAYER).balance(123 * ONE_HUNDRED_HBARS),
+                        cryptoTransfer(tinyBarsFromAccountToAlias(GENESIS, SECP_256K1_SOURCE_KEY, ONE_HUNDRED_HBARS))
+                                .via("autoAccount"),
+                        uploadInitCode(JUST_SEND_CONTRACT),
+                        contractCreate(JUST_SEND_CONTRACT))
+                .when(ethereumCryptoTransferToExplicit(ethBurnAddress, 123)
+                        .type(EthTxData.EthTransactionType.EIP1559)
+                        .signingWith(SECP_256K1_SOURCE_KEY)
+                        .payingWith(RELAYER)
+                        .nonce(0)
+                        .maxFeePerGas(50L)
+                        .maxPriorityGas(2L)
+                        .gasLimit(1_000_000L)
+                        .hasKnownStatus(CONTRACT_EXECUTION_EXCEPTION))
+                .then();
```

Comment:
What does it mean in this test framework when you have a `.then()` with no checks whatsoever?

I wondering why we don't explicitly have to check things like, oh, balance of some account did not change.  Is that just backed into a `.then()` with no checks, that _everything_ must remain unchanged?  (Although that doesn't seem to be the case down at line 558.)

Reply:
Re-arranged this spec to have non-empty `given()`/`when()`/`then()` clauses.